### ISSUE TITLE
using retina waves to produce maps

### ISSUE THREAD

**Issue by dancehours** [ioam/topographica#687]
Hi, I try to  use retina waves in the script of GCAL model to produce maps, e.g. 6000 times. But I can't
get the maps as shown in the J.Neurosci paper (2013). The data looks more like noise. Maybe I misunderstand something ?

**Comment by jbednar** [comment]
Hard to say without more details.  Are you using the code and procedures from https://github.com/ioam/topographica/tree/master/models/stevens.jn13 ?  In particular, the responses to sine-grating stimuli when trained on retinal waves will be weak, so you either need to set the input pattern strength and/or the response thresholds appropriately, or else calculate the map with thresholds turned off (linear activation function).

**Comment by dancehours** [comment]
Yes, I am using that code gcal.ty. Okay, I will try those. Thanks !

**Comment by dancehours** [comment]
Hi Bednar,
by saying "or else calculate the map with thresholds turned off (linear activation function)", which thresholds do you mean? And
what is the linear activation function? Sorry for these questions. Recently I just notice retina wave issue can be investigated further.

**Comment by jbednar** [comment]
Each cortical neuron in LISSOM or GCAL has an activation function. By default this is a piecewise-linear approximation to a sigmoid, with zero outputs for low inputs, increasing values after an initial threshold, and a maximum response after which further input increases have no corresponding output increase. There is thus a linear region by default, with an upper and lower thresholds. If you turn off those thresholds, the output will be linearly related to the input.  In Topographica the default for map measurements is to turn off those thresholds, because otherwise you have to do some research to find suitable input strengths that can drive the output in a reasonable range, i.e. not too weakly (zero output) or too strongly (maximum output). I don't recall which option is used to disable the thresholds, but it's an option set whenever maps are being measured unless you change the defaults.
